gammapy/estimators/map/core.py

```diff
@@ -1093,6 +1105,28 @@ def read(cls, filename, checksum=False):
         ) as hdulist:
             return cls.from_hdulist(hdulist, checksum=checksum)
 
+    def copy(self, reference_model=None):
+        """Deep copy
+
+        Parameters
+        ----------
+
+        reference_model : `~gammapy.modeling.models.SkyModel`, optional
+            The reference model to use for conversions. If None, a model consisting
```

Comment:
The docstring is not correct here. If None, the reference model on self is copied